Experimental/CameraComponent/PerspectiveProjection.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Numerics;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.UI.Composition;
+using Windows.UI.Xaml;
+
+namespace CameraComponent
+{
+    public sealed class PerspectiveProjection : Projection
+    {
+        private float xfov, yfov, near, far;
+        private CompositionPropertySet propertySet;
+
+        public PerspectiveProjection()
+        {
+            xfov = MathF.PI / 2;
+            yfov = MathF.PI / 2;
+
+            near = 1f;
+            far = 1000f;
+        }
+
+        public float XFov
+        {
+            get => xfov;
+            set
+            {
+                xfov = value;
+                RaisePropertyChanged("xFov");
+            }
+        }
+        public float YFov
+        {
+            get => yfov;
+            set
+            {
+                yfov = value;
+                RaisePropertyChanged("yFov");
+            }
+        }
+        public float Near
+        {
+            get => near;
+            set
+            {
+                near = value;
+                RaisePropertyChanged("Near");
+            }
+        }
+        public float Far
+        {
+            get => far;
+            set
+            {
+                far = value;
+                RaisePropertyChanged("Far");
+            }
+        }
+        public event PropertyChangedEventHandler PropertyChanged;
+        public void RaisePropertyChanged(string name)
+        {
+            if (PropertyChanged != null)
+            {
+                PropertyChanged(this, new PropertyChangedEventArgs(name));
+            }
+        }
+        public void CreateExpressionAnimation(CompositionPropertySet toAnimate, string propertyName)
+        {
+            propertySet = Window.Current.Compositor.CreatePropertySet();
+            propertySet.InsertScalar("XFov", xfov);

```

Comment:
It might be nice to have your capitalization of XFov match the capitalization of your xFov in your propertySet.